tests/test_main.py

```diff
@@ -1206,6 +1206,18 @@ def test_sort(dummy: Tensor) -> Tensor:
     return ep.sort(t)
 
 
+@compare_all
+def test_topk_0(dummy: Tensor) -> Tensor:
+    t = ep.arange(dummy, 27).float32().reshape((3, 3, 3))
+    return ep.topk(t, 2)[0]
```

Comment:
Maybe you can also use tuple unpacking here:
`values, _ = ep.topk(t, 2)` and `return values`
(Same for the other test)